### DISCUSSION TITLE
Just a few questions

### DISCUSSION THREAD

**Discussion by LeFede** [tauri-apps/cargo-mobile2#51] [General]
I just wanted to say thank you to the team working on cargo-mobile, and ask a few things. 
I've already installed everything and managed to build to Android using WRY. 

**1.** I don't see tauri template. I have **bevy**, **bevy-demo**, **wgpu**, **winit** & **wry**. Of all those it only worked with **wry**. I'm not sure what all those mean but I saw that in the main repo page there's the tauri template which I don't get to see on my local. Am I doing something wrong?

**2.** After ```cargo mobile update```, **wry** templated dissappeared. Had to install again from the repo to make it work. Both times showed me that cargo mobile version was 0.1.0 . Is this correct?

**3.** I don't see a way to change content on the App. Not sure if is still not allowed to do that or I'm being just dumb. Am I ?

**4.** Without touching anything the app size is ~50MB. Where do this size comes from?

**5.** Do you plan on making a short tutorial video or posting anything concrete anytime soon? 

### Thank you again!

**Comment by wusyong** [comment]
1. cargo-mobile stays as our playground to test integration of mobile. Tauri has it's own CLI to build mobile target. For now, see this [note](https://hackmd.io/@lucasfernog/BkHI4e18j) for more information.
2. I didn't use update command tbh. As I already I treat it more like a playground. If you are interested in fixing it, feel free to open a PR.
3. You can use custom protocol in wry to define what content to represent. We have another [note](https://hackmd.io/XIcEwk4GSxy8APZhSa0UnA?view#Custom-Protocols) for wry.
4. We haven't analyzed yet and we don't have much people to dive deeper to investigate unfortunately. But we welcome and appreciate any effort want to help.
5. Yes, we plan to publish mobile alpha around end of the year. And it should have documentation on tauri.app along the way.
